### ISSUE TITLE
[color-picker] add support for alpha

### ISSUE THREAD

**Issue by jcfranco** [Esri/calcite-components#749]
The color picker needs to support alpha channel updates. https://github.com/Esri/calcite-components/issues/64#issuecomment-583678167 has some designs with opacity.

Not sure if the design needs any tweaks, but one suggestion would be to introduce an alpha channel input as well. The color picker from Chrome dev tool's has something similar: 

![cdt-cp-colors](https://user-images.githubusercontent.com/197440/88196177-c02fb700-cbf5-11ea-9807-058e8fc6a327.gif)

**Comment by bstifle** [comment]
👀

**Comment by jcfranco** [comment]
Another thing to consider in the design is the selected color + transparency preview. One idea would be to update `calcite-color-swatch` to support transparency (rough mockup 😄):

<img width="294" alt="Screen Shot 2020-07-29 at 1 31 12 PM" src="https://user-images.githubusercontent.com/197440/88851391-8f292680-d1a1-11ea-869d-fd76f27daa9b.png">

**Comment by bstifle** [comment]
yeah i dig it

**Comment by macandcheese** [comment]
Yeah looks good... Think that's a pretty universal treatment for opacity so should be familiar.

**Comment by bstifle** [comment]
![image](https://user-images.githubusercontent.com/14083530/88860959-c6073880-d1b1-11ea-925a-8de58e6b135b.png)
here's a comp with the transparency checker fill. 

light theme using 4x4 #000000 squares
dark theme using 4x4 #FFFFFF squares

**Comment by bstifle** [comment]
![image](https://user-images.githubusercontent.com/14083530/88863040-9dce0880-d1b6-11ea-9ae0-6175d300d29d.png)
ok this is better. 

#CACACA for light theme
#6A6A6A for dark

**Comment by smambrose** [comment]
Is there any progress on this one? Testers for charts keep running into it and noticing they can't control this.

**Comment by jcfranco** [comment]
@smambrose Not for the time being. I'll move this to backlog until it's ready for development. I estimate this will land after 1.0.0 ships (after April).

**Comment by tyanthony** [comment]
Having the ability to change the Opacity would be a great enhancement for our team as well. An example of a use case for us: 

![image](https://user-images.githubusercontent.com/17069279/124000773-ec28f000-d9a1-11eb-8b10-45c6ae931ec9.png)

**Comment by julio8a** [comment]
@bstifle, is this one ready for implementation?

**Comment by bstifle** [comment]
Here's the design with an opacity slider:

![image](https://user-images.githubusercontent.com/14083530/128580730-486e856b-aa0c-4f79-89fe-4958701dda5d.png)

**Comment by jcfranco** [comment]
Moving to next sprint. Draft PR should be up today.

**Comment by jcfranco** [comment]
Moving to the next sprint. This should land this week. I've got some review feedback to address and tests to add. 

![dog-so-close](https://user-images.githubusercontent.com/197440/131422816-66a60690-3b6f-4989-bffe-8ddacb1718b3.gif)

**Comment by ethanbdev** [comment]
Hey @jcfranco, was there just a little bit left to do on this? Checking on if it could be for this release cycle, thanks!!

**Comment by phlorah** [comment]
Any update on this?

**Comment by jcfranco** [comment]
Apologies for the belated update. I have an old PR with the functionality in place, but no tests set up. I'll sync it with the latest changes and add the missing tests for the assigned milestone.

**Comment by ashetland** [comment]
Updated designs complete. @geospatialem for visibility. @jcfranco add to Figma file.

**Comment by patrickarlt** [comment]
Is there any expected release date for this? We are waiting on support for alpha and RGBA before migrating Vector Tile Style Editor to this.

**Comment by ashetland** [comment]
@geospatialem Not sure if Franco started work on this before he went on leave.

**Comment by benelan** [comment]
We plan on installing the functionality for v1, however it will go through a design rework afterwards at some point. The design rework would impact layout so it would be a breaking change. [Here is a demo](http://calcite-dev.esri.com:3005/demos/color-picker.html) with the existing functionality if you want to take a look (VPN required)

**Comment by benelan** [comment]
[Here is a publicly accessible link](https://www.chromatic.com/component?appId=6266d45509d7eb004aa254fb&csfId=components-controls-colorpicker&specName=Alpha+Support&buildNumber=310&k=6392837ead8689d0b3d863ab-1200-interactive-true&h=4&b=-1) to the existing `calcite-color-picker` alpha functionality

**Comment by geospatialem** [comment]
Will continue efforts in February's priorities.

**Comment by ashetland** [comment]
Updated designs are available as well.

**Comment by jcfranco** [comment]
Update: this will land this week, but will be available via `next` and not in the April release. Automated tests will be updated/added separately.

**Comment by geospatialem** [comment]
Adding to the new `future breaking change` label, where a few deprecated props take place:
- `hideChannels`
- `hideHex`
- `hideSaved`
